tests/phpunit/Checker/Checks/Enqueued_Scripts_Size_Check_Tests.php

```diff
@@ -94,7 +94,7 @@ public function test_run_with_errors() {
 		$this->assertNotEmpty( $warnings );
 
 		$this->assertEquals( 0, $results->get_error_count() );
-		$this->assertEquals( 4, $results->get_warning_count() );
+		$this->assertEquals( 8, $results->get_warning_count() );
```

Comment:
Why this change? Is there something in our new code that affects this test? If so, we probably have to fix it.

Reply:
@felixarntz The unit tests automatically load all the plugin files, causing all the plugins to be loaded by default. In the commit https://github.com/10up/plugin-check/pull/186/commits/919ec2cd4adffa4d58533eb1720cfe94dcda2792, I removed that functionality to address the issue.

Reply:
@mukeshpanchal27 Good catch, great that you were able to fix that.